packages/fiori/test/samples/UploadCollection.sample.html

```diff
@@ -94,21 +75,22 @@ <h3>UploadCollection</h3>
 				return uci;
 			}
 
-			fileUploader.addEventListener("ui5-change", function(event) {
-				var files = event.detail.files;
+			fileUploader.addEventListener("ui5-change", event => {
```

Comment:
I don't think we transpile samples pages and this will not work on IE.

Reply:
Fixed it, now only the displayed code is in es6